WhaleSpotting/react-app/src/styles/Profile.scss

```diff
@@ -0,0 +1,117 @@
+@import './Constants.scss';
+
+.body {
+    width: 100vw;
+    min-width: $minMobileScreenSize;
+    font-family: $fontFamily;
+    display: flex;
+    flex-direction: column;
+}
+.banner {
+    height: 290px;
+    background-color: $secondaryDark;
+}
+.feed{
+    padding: 0px 20px;
+    flex: 1;
+}
+.little-text{
+    font-size: $smallMobileBodyFontSize;
+    margin: 0px;
+}
+.joined{
+    padding: 0px;
+    margin-top:-7px;
+}
+.reported{
+    margin-left: 5px;
+}
+.feature-text{
+    font-size: $mobileFeatureFontSize;
+    margin:0px;
+}
+.heading{
+    font-size: $mobileTitleFontSize;
+    margin:0px;
+}
+.outer-container{
+    display:flex;
+    flex-direction: row;
+    width: 90vw;
+    margin-left:auto;
+    margin-right:auto;
+}
+.trophy-text-container{
+    display:flex;
+    flex-direction: column;
+}
+.button-container{
+    display:flex;
+    flex-direction: column;
+}
+.trophy-container{
+    display:flex;
+    flex-direction: row;
+    align-items:center;
+}
+.trophy-image{
+    width:70px;
+    height:auto;
+    margin-left:10px;
+    border-radius: 50%;
+}
+.profile-image{
+    width: 160px;
+    height:auto;
+    margin-top:5px;
+    border-radius: 5%;
+}
+.button-container{
+    align-items: center;
+    flex-direction: column-reverse;
+}
+.toggle{
+    background-color: $primaryDark;
+    border-radius:25px;
+    color: $textOnPrimary;
+    font-size:$mobileBodyFontSize;
+    text-decoration: none;
+    width:fit-content;
+    padding:2px 5px;
+    margin:5px;
+}
+@media(min-width: $mobileScreenSize) {
```

Comment:
You are currently styling the desktop with this, I would prefer it if we add the mobile styling with this instead pls :)

Reply:
Styling updated in line with request.